com.unity.ml-agents/Runtime/Timer.cs

```diff
@@ -224,17 +225,22 @@ public string DebugGetTimerString(string parentName = "", int level = 0)
     [DataContract]
     public class GaugeNode
     {
+        static float s_SmoothingFactor = .25f; // weight for exponential moving average.
```

Comment:
const ?

Reply:
changed to `const float k_SmoothingFactor = .25f`